components/Cards/index.js

```diff
@@ -17,3 +17,43 @@
 // </div>
 //
 // Create a card for each of the articles and add the card to the DOM.
+axios
+  .get("https://lambda-times-backend.herokuapp.com/articles")
+  .then(response => {
+    Object.values(response.data.articles).forEach(item => {
+      item.forEach(items => {
```

Comment:
The variable names here are a bit strange.
Doing a `item.forEach(items => {});` typically your plural is on the left of the forEach.
Beyond that I think better names could be used then `item` or `items` in general like `topicArticles` and `article`